templates.js

```diff
@@ -382,6 +382,894 @@ module.exports = {
     default: "paymentData.shipping"
     }, 
    ]
+  },
+  {
+    id: 'CdpPopup',
+    name: 'CDP Simple Popup',
+    description: `
+    This is a simple popup template which can be used in a popup tag for CDP scenarios. For more information please have a look at this article: https://help.piwik.pro/support/cdp/6-useful-examples-of-how-to-use-audiences-and-data-activations/
```

Comment:
I think this url should work on click. Here it is just plain text. Also I discussed it with Bartek and we think that 
https://help.piwik.pro/support/tag-manager/custom-pop-up-tag/ would be helpful too

Reply:
Addressed in the latest commit. I think it would be worth to sanitize the HTML in the future, e.g. [DOMPurify](https://github.com/cure53/DOMPurify).

Reply:
Just to let u know that  `dangerouslySetInnerHTML` usage opens up XSS security issue but we already had it in `template.beforeDescriptionNote` so it not something new but still not a way to go, but for now I don't see easy-pick solution for that ;/

Reply:
@rezik007 wouldn't DOMPurify at leasy partiall solve this?

Reply:
Maybe but thats not crucial for now imho, we can leave it as it is.